data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -117,7 +117,7 @@ def _increment_total_eligibility_exceeded(
         """
 
         self.df[field_names.THRESHOLD_COUNT] += self.df[columns_for_subset].sum(
-            axis=1
+            axis=1, skipna=True
```

Comment:
Doesn't change any logic; skipna is True by default. I was investigate weird differences that turned out to be unrelated to this.

Reply:
I would be curious to know what the differences were.